src/kakarot/gas.cairo

```diff
@@ -55,6 +55,7 @@ namespace Gas {
     const TX_ACCESS_LIST_ADDRESS_COST = 2400;
     const TX_ACCESS_LIST_STORAGE_KEY_COST = 1900;
     const BLOBHASH = 3;
+    const MAX_MEMORY_SIZE = 0x1cdbe1;  // for a block gas limit of 7M
```

Comment:
update the `MEMORY_COST_U128` to be the associated memory cost (ie 7000130) and rename